internal/builder/builder.go

```diff
@@ -358,10 +358,15 @@ func (b *Builder) Save(logger logging.Logger, creatorMetadata CreatorMetadata) e
 		return errors.Wrap(err, "adding stack.tar layer")
 	}
 
+	if len(b.env) > 0 {
+		logger.Debugf("Environment variables available are %s", style.Map(b.env))
```

Comment:
Minor wording change:

```suggestion
		logger.Debugf("Provided Environment Variables %s", style.Map(b.env))
```


I'm thinking of cases where there could be more environment variables. I'm not sure that inlining them would be the most readable.

Could we make them be printed out in individual lines (and indented)? 

For example:

```
Provided Environment Variables:
  ENV_1=value
  FOO=bar
  BAR=1
```

Reply:
yeah i agree with you, i updated the PR. now it shows 
```
-> google.python.missing-entrypoint@0.9.0
-> google.utils.label@0.0.1
Provided Environment Variables
  BAR=bar
  FOO=bar
```
Also added separator & prefix to `Map` func so it can be reused somewhere else. may be with a different way of showing key value pairs.